public/rest-page.js

```diff
@@ -0,0 +1,67 @@
+// splitting the search part of the url at '=' and then slicing to get the only element of the array that's left which is the restaurant's id
+let restaurantId = window.location.search.split("=").slice(-1)[0]
+
+// DOM scripting to get each element by their id
+let restDisplay = document.getElementById("restaurant")
+let restName = document.getElementById("restName")
+let restAddress = document.getElementById("restAddress")
+let restPhone = document.getElementById("restPhone")
+let restHours = document.getElementById("restHours")
+let restMap = document.getElementById("restMap")
+
+
+let notes = restaurant.notes
+let name = restaurant.name
+
+// doesn't work, need to fix. it's supposed to iterate over each comment and put them into a list
+function custComments(comments) {
+  comments.forEach((comment) => {
+    let custComment = document.createElement("li")
+    custComment.textContent = comment
+    commentsList.appendChild(custComment)
+  })
+}
```

Comment:
There's a few things going on between here, line 46 and line 51 that are a bit conflicting and causing these to not work how you expect them to be